src/collar/oc_titler.lsl

```diff
@@ -239,7 +239,7 @@ UserCommand(integer iAuth, string sStr, key kAv) {
             } else {
                 g_iOn = TRUE;
                 if (sCommand == "graph") {
```

Comment:
I would suggest to test more here if this is meant to be a graph by:
`if (sCommand == "graph" && ( sAction == "0" || (integer)sAction) {`